aws-cli-assumerole.sh

```diff
@@ -1,28 +1,47 @@
 #! /bin/bash
```

Comment:
@letic just wanted to say thanks for opening this PR ❤️

In short: your changes are welcomed and improve this bash script considerably.